tests/test_filtering.py

```diff
@@ -385,6 +385,14 @@ class PostFilter(FilterSet):
         self.assertEqual(f.__module__, 'tests.test_filtering')
         self.assertEqual(f.__name__, 'LocalTagFilter')
 
+    def test_empty_param_name(self):
+        GET = {'': 'foo', 'author': User.objects.get(username='user2').pk}
+        try:
+            f = NoteFilter(GET, queryset=Note.objects.all())
+        except IndexError:
+            self.fail()
+        self.assertEqual(len(list(f.qs)), 1)
+
```

Comment:
The `try`/`except` isn't necessary here, since the `IndexError` is unexpected and will cause the test to error.

Reply:
Removed.